components/Form.js

```diff
@@ -0,0 +1,86 @@
+import Button from "./Button";
+import { collections } from "@/lib/data";
+import styled from "styled-components";
+import { uid } from "uid";
+import { useState } from "react";
+
+const StyledFormContainer = styled.div``;
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  padding: 1em 2em;
+`;
+
+const SmallText = styled.p`
+  font-size: 0.8rem;
+`;
+
+const SubmitMessage = styled.p`
+  text-align: center;
+`;
+
+export default function Form({ onAddFlashCard }) {
+  const [isCardCreatedMessage, setIsCardCreatedMessage] = useState("");
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    setIsCardCreatedMessage("New Card Created.");
+
+    const formElements = event.target.elements;
+
+    const formAnswer = formElements.answer.value;
+    const formQuestion = formElements.question.value;
+    const selectedCollectionTitle = formElements.collections.value;
+
+    const collectionId =
+      collections.find(
+        (collection) => collection.title === selectedCollectionTitle
+      )?.id || null;
+
+    const newFlashCard = {
+      id: uid,
```

Comment:
To create an id with uid, you need to call it:

```suggestion
      id: uid(),
```